### DISCUSSION TITLE
!roll replacement idea

### DISCUSSION THREAD

**Discussion by Avelia-tr** [ppy/osu#29633] [Ideas]
!roll is a staple of the tournament scene so I think it would be cool to have some way to get that in osu. 
One idea I have was to copy the way baldur's gate 3 do the dice roll cuz it looks really cool 
![nat-20-baldur's-gate](https://github.com/user-attachments/assets/e8eb3e7f-deed-4819-b203-6299da3cb4a4)
it would just be a button like the start button but for each player that would show this little animation on the client and post the result in chat

**Comment by Tenexxt** [comment]
would be cool in tournaments in lazer if it had an animation, tho i think there are more important things to be made for lazer tournaments
